terraform/modules/envoy-configuration/main.tf

```
@@ -0,0 +1,77 @@
+# See the user guide at
+# https://docs.aws.amazon.com/app-mesh/latest/userguide/app-mesh-ug.pdf
+# for more details on configuring Envoy in AppMesh
+
+variable "mesh_name" {
+  type = string
+}
+
+variable "service_name" {
+  description = "Service name of the Fargate service, cluster, task etc."
+  type        = string
+}
+
+variable "log_group" {
+  type = string
+}
+
+variable "aws_region" {
+  type = string
+}
+
+locals {
+  # From the user guide: Our examples use 1337 for historical purposes.
```

I see this is indeed a direct quote from the App Mesh user guide, but the malapropism _historical purposes_ is a bit confusing (at least to my mind). They presumably meant historical _reasons_. How about
```suggestion
  # 1337 is an arbitrary choice copied from the examples in the user guide.
```